bigquery_etl/cli/query.py

```diff
@@ -1578,8 +1578,6 @@ def deploy(
             click.echo(f"No schema file found for {query_file}")
             continue
 
-        click.echo(f"Deploying schema for {query_file}")
```

Comment:
Removing this since the message might be confusing since the schema might not actually get deployed (e.g. when using `--skip-existing`)